From 38ea965869a436c4b60e7dbfd5a28d42232d82d6 Mon Sep 17 00:00:00 2001
From: Fabio Milano
Date: Thu, 13 Aug 2015 16:03:26 +0200
Subject: [PATCH] Added no-strict parameter

The command line tool “codesign” has changed in 10.9.5 and 10.10, you need to
pass “--no-strict” option to the command, (the problem has been reported and
will be fixed). To workaround the problem, please save a copy and modify
PackageApplication to pass “—no-strict” to codesign, you can locate PackageApplication
by running the following:- xcrun -sdk iphoneos -f PackageApplication

See http://stackoverflow.com/questions/26008449/xcodebuild-codesign-vvvv-saysresource-envelope-is-obsolete

---
 PackageApplication4Gym | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/PackageApplication4Gym b/PackageApplication4Gym
index 3085608..54a4b9e 100755
--- a/PackageApplication4Gym
+++ b/PackageApplication4Gym
@@ -92,7 +92,7 @@ foreach $plugin (@plugins) {
     chomp $pluginName;
     my $destPlugin = "$destAppDir/$pluginName";
     
-    my $result = runCmd("/usr/bin/codesign", "--verify", "-vvvv", , $plugin );
+    my $result = runCmd("/usr/bin/codesign", "--verify", "--no-strict", "-vvvv", , $plugin );
     if ( $result !~ /valid on disk/ ) {
         fatal("Codesign check fails : $result\n");
     }
@@ -111,7 +111,7 @@ if ( $opt{symbols} ) {
 
 if ( $opt{verbose} ) {
     print "### Checking original app\n";
-    my $result = runCmd("/usr/bin/codesign", "--verify", "-vvvv", , $origApp );
+    my $result = runCmd("/usr/bin/codesign", "--verify", "--no-strict", "-vvvv", , $origApp );
     if ( $result !~ /valid on disk/ ) {
         print "Codesign check fails : $result\n";
     }
-- 
2.2.1

